devDocs/startupShutdownInstall.md

```diff
@@ -0,0 +1,82 @@
+# NVDA Starting and Exiting outline
+
+## Ways to start NVDA:
+
+1. For an installed copy:
+    1. Ctrl+Alt+N (Desktop shortcut)
+    1. Automatically via Ease of Access on the Windows sign-in screen (at boot or signing out of a previous session)
+    1. Automatically via Ease of Access on User Account Control (UAC) screens
+    1. Automatically by Ease of Access after signing in to Windows
+1. For an installed copy, portable copy, installer:
+    1. An exiting instance of NVDA starting a new process (see shutting down procedures)
+    1. By running the exe (see cli options too).
+    This can be triggered by a user or external process such as an existing NVDA instance
+    - test: `startupShutdownNVDA.Starts`
+1. For source: eg runnvda.bat

```

Comment:
I think this is a good overview. I'd also like a "test plan" for each of these (perhaps adding a new heading in a new "test plan" section for each of these. Test plans are particularly useful for demonstrating the edge cases around updating, and providing the detailed steps and expectations for a manual tester. If we have to do manual testing, this can save a lot of time by spelling out how to test, especially for the trickier use cases. Ideally when someone is modifying this area of NVDA we can say follow this test plan to confirm everything still works.